android/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What are these files?!

Reply:
I checked with our android guy, and it seems this should be in the `.gitignore` file. I will add it.